### PR TITLE
adjusted for v.1.4.6

### PR DESCRIPTION
**Pull request by jaecktec**

### CODE REVIEW

**Comment by jamesward** [comment]
It looks like the spaces have become tabs.  Can you fix that?

**Comment by jaecktec** [comment]
Did this fix it?

File: pom.xml [hunk 1]
```diff
@@ -11,18 +12,36 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>textAngular</artifactId>
-    <version>1.4.2-SNAPSHOT</version>
+    <version>1.4.6</version>
     <name>textAngular</name>
     <description>WebJar for textAngular</description>
     <url>http://webjars.org</url>
 
-    <developers>
```
Comment (jamesward) [inline pom.xml:19]: This section should stay and you can add yourself if you want.

File: pom.xml [hunk 2]
```diff
@@ -98,6 +194,12 @@
                 </configuration>
             </plugin>
         </plugins>
+        <resources>
```
Comment (jamesward) [inline pom.xml:197]: This section is no longer needed.

File: pom.xml [hunk 3]
```diff
@@ -11,18 +12,36 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>textAngular</artifactId>
-    <version>1.4.2-SNAPSHOT</version>
+    <version>1.4.6</version>
```
Comment (jamesward) [inline pom.xml:15]: This should be `1.4.6-SNAPSHOT` since Maven does the actual release.

**Pull request closed without merging**